### DISCUSSION TITLE
Maid speed

### DISCUSSION THREAD

**Discussion by samolego** [Mobile-Artificial-Intelligence/maid#282] [General]
Hello there! I am developing an android-only app using java-llama.cpp bindings, and currently I am getting quite poor performance compared to your app (text gen is quite slow).

Did you do any special optimizations / build llama.cpp with clblast & openblas?
Thanks :)

**Comment by danemadsen** [comment]
Hi, sorry i didn't see this discussion. I'm not using any optimizations for the app, i tried adding in clblast and openblast support but it caused the app to become unstable.  It could just be that dart and C++ are faster languages

**Reply by samolego** [reply]
Thanks for the info

**Reply by samolego** [reply]
I was testing the debug speed :/, release one is similar to maid :)
